src/core/HKXWrangler.cpp

```diff
@@ -1961,6 +1964,12 @@ hkGeometry extract_bounding_geometry(FbxNode* shape_root, set<pair<FbxAMatrix, F
 	}
 	if (properties.m_mass == 0.0)
 	{
+		const auto property_mass = get_property<float>(shape_root, "mass", 0.f);
```

Comment:
Added ability to set mass and inertia tensor thorugh custom properties for case when computed values are off.